Library/Homebrew/test/dev-cmd/test_spec.rb

```diff
@@ -7,7 +7,7 @@
 end
 
 # randomly segfaults on Linux with portable-ruby.
-describe "brew test", :integration_test, :needs_macos do
+describe "brew test", :integration_test, :needs_macos, timeout: 120 do
```

Comment:
This definitely shouldn't be taking anywhere near the timeout? I wonder what's going on here.

Reply:
No idea, I saw it spuriously fail yesterday.